src/App.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from "react";
+import { Matrix } from "./components/MatrixBlock";
+import { MatrixBuilder } from './components/MatrixBuilder';
+import styles from './index.scss';
+
+function App() {
+  const [data, setData] = useState({
+    numberRows: 1,
+    numberColumns: 1,
+    disable: "",
+    arr: [],
+    sum: 0,
+  })
+
+  const [matrixData, setMatrixData] = useState({
+    rows: 0,
+    columns: 0,
+    cells: 0,
+  })
+
```

Comment:
let's create handleSetMatrix = (data) => setData(data) and pass it as props onSetMatrix

Reply:
Why do i need to do it?